### PR TITLE
Move cursor pagination to the UI layer and minor cosmetic changes

### PR DESCRIPTION
**Pull request by rufusu-sony**
- Cursor pagination URLs are now passed to the UI layer.
- This removes the work around requiring random numbers to be generated to cause an update.
- The `<PaginationButton>` component has been refactored
- Fix floating tabs in Chrome
- Add spacing and raise transport parameter cards

### CODE REVIEW

**Comment by rufusu-sony** [comment]
This is fully functional however there is still duplicated code. Would it be worth refactoring List views to higher level components for this pull request?
Ultimately what I've been thinking of is the same level of abstraction found in the react-admin `<DataGrid>` and `<List>` components where we can build views like so:
```jsx
const SendersList = props => (
    <List {...props}>
        <Datagrid>
            <TestField source="label" />
            <TextField source="transport" />
            <TextField source="subscription.active" />
        </Datagrid>
    </List>
);
```

**Comment by garethsb-sony** [comment]
Looking good, and yes, I think that should be ultimate goal, but should be second step. Let's review this in detail early next week. 👍

File: Development/src/pages/devices/DevicesList.js [hunk 1]
```diff
@@ -98,10 +101,10 @@ const DevicesList = props => {
                         </TableBody>
                     </Table>
                     <br />
-                    <PaginationButton label="FIRST" nextPage={nextPage} />
-                    <PaginationButton label="PREV" nextPage={nextPage} />
-                    <PaginationButton label="NEXT" nextPage={nextPage} />
-                    <PaginationButton label="LAST" nextPage={nextPage} />
+                    <ListPaginationButton label="first" />
```
Comment (garethsb-sony) [inline Development/src/pages/devices/DevicesList.js]: I'd like to see this as `<ListPaginationButton label="First" rel="first"/>` with `label` defaulted from `rel` in the same way as other React-Admin components default their label from `source` or `name` props. Is that doable quickly?

Comment (garethsb-sony) [reply Development/src/pages/devices/DevicesList.js]: And I think you're right that defining a `<Pagination/>` component that created all four (or two, if no first/last cursors provided) buttons would be a good step. Even if we don't fully fit back into the React-Admin model, https://github.com/marmelab/react-admin/blob/master/packages/ra-ui-materialui/src/list/Pagination.js could provide inspiration for the interface?

**Comment by rufusu-sony** [comment]
I also have a couple very minor changes made on my [Development Branch](https://github.com/rufusu-sony/nmos-js/commits/development)
This includes:
- Fixing the QueryAPI connect button as it is non functional
- Improving docker build times with a .dockerignore file

Is it a good idea to bundle these with this pull request or would you like a follow up request after this is merged?

**Comment by garethsb-sony** [comment]
Thanks, Rufus. I incorporated the other changes from dev. :+1:

**Pull request merged**